includes/distributor-customizations/global.php

```diff
@@ -1,23 +1,82 @@
 <?php
 /**
- * This files contains the global tweaks that are loaded by default in every site in the Network (bot Hub and Nodes)
+ * This files contains the global tweaks that are loaded by default in every site in the Network (both Hub and Nodes).
  *
  * @package newspack-network
  */
 
 /**
- * =========================================
- * ======== Post publication date ==========
- * =========================================
- * Keep the publication date on the new pushed post.
+ * Send primary category slug to the Node along with post update data.
  *
- * This filter is used to filter the arguments sent to the remote server during a push. The below code snippet passes the original published date to the new pushed post and sets the same published date instead of setting it as per the current time.
+ * @param WP_Post $post The post object.
+ * @param bool    $is_pulling Whether the post is being pulled from the remote site.
+ */
+function newspack_network_get_primary_category_slug( $post, $is_pulling = false ) {
+	if ( $is_pulling ) {
+		// When pulling content, the post will be the Node post.
```

Comment:
The "Hub" and "Node" term don't apply here. Posts can be distributed from any site to any site (node to node, node to hub, etc).

It's better to say the origin and target sites, or something like that